mesonbuild/templates/cythontemplates.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2019 The Meson development team
+
+from __future__ import annotations
+
+from mesonbuild.templates.sampleimpl import FileImpl
+
+import typing as T
+
+
+hello_pyx_template = '''print("This is project {project_name}.")'''
+
+hello_pyx_meson_template = '''project('{project_name}', 'python',
```

Comment:
Has this been tested? Because this isn't a valid language... it's not the language of the template itself...